src/shared/containers/header-container.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Created by Youngchan Je on 2016-09-05.
+ */
+import React from 'react';
+import {connect} from 'react-redux';
+import Header from '../components/header';
+
+const App = ({children}) => {
```

Comment:
I separated the component!
well, I put the component in the MapLayout and deleted header-container since it is exactly same as app-container without header component